NitroxModel/NitroxModel.csproj

```diff
@@ -10,7 +10,7 @@
         <PackageReference Include="LiteNetLib" Version="1.2.0" />
         <PackageReference Include="Microsoft.Win32.Registry" Version="5.0.0" />
         <PackageReference Include="Mono.Nat" Version="3.0.2" />
-        <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+        <PackageReference Include="Newtonsoft.Json" Version="12.0.3" NoWarn="NU1091;NU1092;NU1903;NU1904" />
```

Comment:
What was the reason that we can't upgrade past this? I am going to assume it clashes with something else?

Reply:
We're using subnautica dll which is 12.0.3 to avoid conflict afaik, for some reasons UWE didn't want to upgrade their packages while upgrading their game

Reply:
Eventually we'll move everything to System.Text.Json when the linux branch will come